airflow/models/dagbag.py

```diff
@@ -248,7 +247,7 @@ def process_file(self, filepath, only_if_updated=True, safe_mode=True):
 
             with timeout(self.DAGBAG_IMPORT_TIMEOUT):
                 try:
-                    m = imp.load_source(mod_name, filepath)
+                    m = importlib.machinery.SourceFileLoader(mod_name, filepath).load_module()
```

Comment:
```suggestion
                    m = importlib.machinery.SourceFileLoader(mod_name, filepath).exec_module()
```

https://docs.python.org/3/library/importlib.html#importlib.machinery.SourceFileLoader.load_module

Reply:
Actually you then also need to call https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.create_module first.

Reply:
Thanks. I'll fix it.